### PR TITLE
Updated documentation with api information

### LINKED ISSUE #182: Improve setup docs
### 🐛 Describe the bug

The setup documentation currently refers to creating a `.dev.env` file, but the project actually uses a `.env` file. Additionally, there are no mentions of the APIs being used or explanations of why they are needed.

Update the documentation to accurately reflect details about the `.env` file and the API keys

### PR DESCRIPTION
**Pull request by lakshyasarve**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Refactor (does not change functionality, e.g. code style improvements, linting)
- [x] Documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

Please include screenshots below if applicable.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

## Maintainer Checklist

- [x] closes #182
- [x] Tag the PR with the appropriate labels

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,11 +41,27 @@ Install dependencies
 
 Create `.env` using template `.env.template` and add API keys
 ```
-cat .env.template > .dev.env
+cat .env.template > env
```
Comment (mdmohsin7) [inline README.md]: Please change `env` to `.env`

**Review by mdmohsin7 — CHANGES_REQUESTED** [review]

**Comment by lakshyasarve** [comment]
Completed the suggested changes

File: README.md (hunk 1 above)
Comment (lakshyasarve) [reply README.md]: Done!

**Review by mdmohsin7 — APPROVED** [review]
lgtm

**Pull request merged**